web/src/features/moreCast2/interfaces.ts

```diff
@@ -46,8 +46,9 @@ export interface MoreCast2Row extends BaseRow {
   fwiCalcForecast?: PredictionItem
   dgrCalcForecast?: PredictionItem
 
-  // Grass curing carryover
-  grassCuring: number
+  // Grass curing
+  grassCuringActual: number
```

Comment:
Is `grassCuringActual` required because of the way things are setup?Conceptually only estimates/forecasts of grass curing exist.

Reply:
Good question, I debated over using Actual/Forecast even though it's a bit misleading. Completely agree, it's not even truly a forecast, it's just submitted with a forecast
I ended up going this route to reuse logic for things like `valueSetter`s, disabling cells, and some other logic in the cells. I also thought it might be useful to reuse `PredictionItem` if we want to use the CWFIS 'model' as a model choice. Though I suppose something other than a `PredictionItem` could be created fairly easily. 

Open to changing this if we want to